### ISSUE TITLE
Security scan issues for fpm containers

### ISSUE THREAD

**Issue by Den4ik** [davidalger/warden#99]
https://quay.io/repository/warden/php-fpm?tab=tags

**Comment by davidalger** [comment]
@Den4ik Good call out. :) Not super important with the FPM images specifically being used here since they are targeting development use rather than deployments, nevertheless the scanning Quay has is one of the reasons I shifted images over there. I wanted to be able to see this data, but hadn't gotten around to doing anything with it yet.

In the case of the FPM images though, the stuff showing up there was due to outdated packages in the most recent CentOS 7 base image (easily resolved by adding a yum update to the top of the dockerfile) and due to erroneously having the php-devel pkg installed which brought in kernel headers (which are always going to flag meaninglessly). I took care of this in these commits in the upstream php images these FPM images rely on (where the resulting images are in fact meant for and used in production scenarios):

* https://github.com/davidalger/docker-images-php/commit/62ecd91c8b3f61985ad09b4278891478eb424484
* https://github.com/davidalger/docker-images-php/commit/2e381f6fb2a1bbef6430ffcc126383ded561658a

![warden_php-fpm_·_Quay_-_Mozilla_Firefox](https://user-images.githubusercontent.com/658281/74794074-d8d32400-5287-11ea-860e-7ee12165e85f.png)

It is worth noting some of the other images in the Warden namespace on Quay are going to show vulnerabilities due to them being mirrors of official images (that trip the scanners) or in the case of Varnish where kernel-headers is (for some reason) required by the varnish packages and so I cannot get rid of them just to get a green scan on those.

Thanks again for the note.
